webextension/css/context.css

```diff
@@ -43,6 +43,14 @@ h4 {
     color: #333;
 }
 
+.sidebar-tabs div:first-child button {
+    border-radius: 5px 5px 0 0 !important;
+}
+.sidebar-tabs div:last-child button {
+    border-radius: 0 0 5px 5px !important;
+}
```

Comment:
Thank you for this @cgorringe 
This always adds the borders to the first and last tabs (Alexa and Tag Cloud) written in HTML markup, irrespective of the first and last tab that is displayed in the Menu. I made some JS modifications to achieve the desired behavior. Please check :-)